Lesson2/src/Lesson2/Main.java

```diff
@@ -0,0 +1,173 @@
+// @author Shkarin Konstantin
+
+package Lesson2;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+public class Main {
+
+    public static void main(String[] args) {
+        // сортировка Bubble подсчет времени массива arr
+        MyArray arr = new MyArray(1000);
+        for (int i = 0; i < 1000; i++) {
+            int r = (int) (Math.random()*arr.getSize());
+            arr.add(r);
+        }
+        long start = System.nanoTime();
+        arr.sortBubble();
+        long finish = System.nanoTime();
+        long delta = finish-start;
+        TimeUnit.NANOSECONDS.toMillis(delta);
```

Comment:
метод toMillis возвращает форматированный результат, который и следует использовать. Всегда обращайте внимание на javadoc методов, что они принимают, делают и возвращают
```suggestion
        delta = TimeUnit.NANOSECONDS.toMillis(delta);
```